src/pperm.c

```diff
@@ -341,7 +336,7 @@ static Obj PreImagePPermInt(Obj pt, Obj f)
 * GAP functions for partial perms
 *****************************************************************************/
 
-
+// FIXME why is this a function???
 Obj FuncEmptyPartialPerm(Obj self)
```

Comment:
Alas, whether there is a good justification or not, this function is documented in the GAP reference manual, so we cannot just remove it. Of course it is also documented as a "short hand for PartialPerm([]);." So one could replace this C function by a GAP function. And then, if you felt strongly about it, also mark it as obsolete.

Not sure that new C comment is going to help anybody, though ;-).